chat-widget/src/components/livechatwidget/common/authHelper.ts

```diff
@@ -25,15 +26,23 @@ const handleAuthentication = async (chatSDK: any, chatConfig: ChatConfig | undef
             (chatSDK as any).setAuthTokenProvider(async () => {
```

Comment:
type

Reply:
this seems to be a hack, since setAuthTokenProvider is private, I need to investigate why this hack is needed, Ill create a task